src/views/profile/edit.ejs

```diff
@@ -93,14 +93,14 @@
         <option value="BJ">Benin</option>
         <option value="BM">Bermuda</option>
         <option value="BT">Bhutan</option>
-        <option value="BO">Bolivia, Plurinational State of</option>
+        <option value="BO">Bolivia</option>
         <option value="BQ">Bonaire, Sint Eustatius and Saba</option>
         <option value="BA">Bosnia and Herzegovina</option>
         <option value="BW">Botswana</option>
-        <option value="BV">Bouvet Island</option>
```

Comment:
This doesn't have an addition, did you mean to completely remove it?

Reply:
Oops